mopro-ffi/README.md

```diff
@@ -46,3 +46,10 @@ To test bindings:
 To test bindings in release mode without warning:
 
 `cargo test --test test_generated_bindings --release 2>/dev/null`
+
+## Generate MSM benchmark report in swift on laptop
+
+* Enable `gpu-benchmarks` feature as default in `Cargo.toml`
```

Comment:
To clarify, after adding feature flag around test_generate_bindings.rs this is still needed? Can't just pass flag as argument?

Reply:
No, we cannot. The feature flag for now doesn't work though, we can only enable the feature through the `deafult` flag in `Cargo.toml`